packages/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -426,6 +424,7 @@ describe('plugin-meetings', () => {
             audioContext: {},
             mode: 'WORKLET',
             env: 'prod',
+            avoidSimd: false,
```

Comment:
suggestion: These are all the current default values - in order to prove that the custom options are being used, they should be non-default values.